### PR TITLE
Data page updates

### PR DESCRIPTION
**Pull request by dissco6**
Front-end side data page updates

### CODE REVIEW

File: front-end-side/src/Pages/EnterData.js [hunk 1]
```diff
@@ -42,11 +43,16 @@ const ShowData = () => {
             if (!dateRegEx.test(date) && isNotEmptyRegEx.test(date)) {
                 addErrorBorder("date");
                 printErrorMsg("Date is invalid or field is empty. Date format should be <b>xxxx-xx-xx</b>");
+            } else if (inputDate > currentDate && isNotEmptyRegEx.test(date)) {
+                addErrorBorder("date");
+                printErrorMsg("Date is later then current date");
```
Comment (Vaivaal) [inline front-end-side/src/Pages/EnterData.js:48]: should be "than" instead of "then"

Comment (dissco6) [reply front-end-side/src/Pages/EnterData.js]: Fixed

File: front-end-side/src/Pages/FileUpload.js [hunk 2]
```diff
@@ -25,40 +32,78 @@ const FileUpload = () => {
             const res = await axios.post("https://localhost:7174/api/Upload", newData, {
                 headers: {
                     "Content-Type": "multipart/form-data"
-                },  
+                },
             }).then(res => {
                 console.log(res);
             });
-            console.log(res); 
+            console.log(res);
```
Comment (shilingas) [inline front-end-side/src/Pages/FileUpload.js:39]: there is no point of console log

Comment (dissco6) [reply front-end-side/src/Pages/FileUpload.js]: I removed them

**Pull request merged**